### ISSUE TITLE
The `os` explorer categorizes `guixsd` too eagerly

### ISSUE THREAD

**Issue by lunarnull** [skonfig/base#96]
The `os` explorer checks for the existence of `/gnu/store` directory.  This directory is not only present on Guix Systems, but anywhere Guix (package manager) is installed.  The practice of installing Guix (package manager) on top of other distros is [explicitly encouraged](https://guix.gnu.org/en/manual/en/html_node/Installation.html), leading to mischaracterization of distros which are checked further in the script.  For example, on my PopOS box:

```console
$ ./os
guixsd
$ (. /etc/os-release && echo $NAME $ID_LIKE)
Pop!_OS ubuntu debian
```

I don't know enough about Guix System to suggest a better fix than moving the `guixsd` check further down the script.

**Comment by sideeffect42** [comment]
Thank you for your bug report, @lunarnull.

I will see if I can find a better way to detect Guix and replace the `/gnu/store` check. If not, we should certainly move this check further down.

Also I'm noticing we don't have a check for Pop! OS yet :-)

**Comment by sideeffect42** [comment]
@lunarnull This bug should be fixed by #97. Can you please confirm if these changes fix the bug you noticed?

Thanks.

**Comment by sideeffect42** [comment]
> I think the `in` part of [the one `case` statement](https://github.com/skonfig/base/pull/97/commits/96531da2c90f160f550a21bcfd9c8b15d7adee3f#diff-937d4c5c2de5d4d6588200c83cb66cbc542fb24d6dfed90baf01c4354471e2d4R180) got accidentally removed.

Ooops. Fixed it :-)
